### PR TITLE
Adds missing return

### PR DESCRIPTION
**Pull request by gstark**

### CODE REVIEW

File: lib/formatters/ruby.js [hunk 1]
```diff
@@ -18,12 +18,16 @@ module.exports = function formatRuby(code, filepath) {
 
   if (configPath !== null) {
     configOption = `--config ${configPath}`;
+  } else {
+    configOption = "";
```
Comment (rposborne) [inline lib/formatters/ruby.js]: Yup... missed that edge case.

File: lib/formatters/ruby.js [hunk 2]
```diff
@@ -18,12 +18,16 @@ module.exports = function formatRuby(code, filepath) {
...
   }
 
   fs.writeFileSync(tmpobj.name, code);
   try {
+    let shellCommand = `rubocop -f j ${configOption} --auto-correct --stdin ${tmpobj.name}`
```
Comment (rposborne) [inline lib/formatters/ruby.js]: Love this.  Maybe a const? this value should not change in the scope of this function.

**Review by rposborne — APPROVED** [review]
Thanks for this! Looks good.

File: lib/formatters/ruby.js (hunk 2 above)
Comment (gstark) [reply lib/formatters/ruby.js]: You are correct sir. I keep forgetting to default to `const`

**Comment by rposborne** [comment]
Looks great!

**Pull request merged**